GRAMMAR.md

```diff
@@ -108,7 +108,7 @@ on filtering values
 ```
      condition
        : literal
-       | function
+       | literal_function
```

Comment:
Note, that this will prevent fields from being included on the right hand side of an expression syntactically.